com/williamfiset/algorithms/strings/BoyerMooreStringSearch.java

```diff
@@ -0,0 +1,51 @@
+package com.williamfiset.algorithms.strings;
+
+import static java.util.Objects.isNull;
+
```

Comment:
Can you update the README string section to add an entry for Boyer-Moore to point to this file?